<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>905</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Character Review</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="main" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1351</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sheetBtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>630</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>View Character Sheet</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chrComboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>637</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="histogram" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>1351</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="archTitle">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>630</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Optimising:</string>
    </property>
   </widget>
   <widget class="QLabel" name="archLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>630</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1355</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
